docs/ops/sort/MulticlassNMS_9.md

```diff
@@ -0,0 +1,175 @@
+The total number of output boxes of each batch element must not exceed `keep_top_k`.
+Boxes of `background_class` are skipped and thus eliminated.
+
+**Attributes**:
+
+* *sort_result*
+
+  * **Description**: *sort_result* specifies the order of output elements.
+  * **Range of values**: `class`, `score`, `none`
+    * *class* - sort selected boxes by class id (ascending).
+    * *score* - sort selected boxes by score (descending).
+    * *none* - do not guarantee the order.
+  * **Type**: `string`
+  * **Default value**: `none`
+  * **Required**: *no*
+
+* *sort_result_across_batch*
+
+  * **Description**: *sort_result_across_batch* is a flag that specifies whenever it is necessary to sort selected boxes across batches or not.
+  * **Range of values**: true or false
+    * *true* - sort selected boxes across batches.
+    * *false* - do not sort selected boxes across batches (boxes are sorted per batch element).
+  * **Type**: boolean
+  * **Default value**: false
+  * **Required**: *no*
+
+* *output_type*
+
+  * **Description**: the tensor type of outputs `selected_indices` and `valid_outputs`.
+  * **Range of values**: `i64` or `i32`
+  * **Type**: `string`
+  * **Default value**: `i64`
+  * **Required**: *no*
+
+* *iou_threshold*
+
+  * **Description**: intersection over union threshold.
+  * **Range of values**: a floating-point number
+  * **Type**: `float`
+  * **Default value**: `0`
+  * **Required**: *no*
+
+* *score_threshold*
+
+  * **Description**: minimum score to consider box for the processing.
+  * **Range of values**: a floating-point number
+  * **Type**: `float`
+  * **Default value**: `0`
+  * **Required**: *no*
+
+* *nms_top_k*
+
+  * **Description**: maximum number of boxes to be selected per class.
+  * **Range of values**: an integer
+  * **Type**: `int`
+  * **Default value**: `-1` meaning to keep all boxes
+  * **Required**: *no*
+
+* *keep_top_k*
+
+  * **Description**: maximum number of boxes to be selected per batch element.
+  * **Range of values**: an integer
+  * **Type**: `int`
+  * **Default value**: `-1` meaning to keep all boxes
+  * **Required**: *no*
+
+* *background_class*
+
+  * **Description**: the background class id.
+  * **Range of values**: an integer
+  * **Type**: `int`
+  * **Default value**: `-1` meaning to keep all classes.
+  * **Required**: *no*
+
+* *normalized*
+
+  * **Description**: *normalized* is a flag that indicates whether `boxes` are normalized or not.
+  * **Range of values**: true or false
+    * *true* - the box coordinates are normalized.
+    * *false* - the box coordinates are not normalized.
+  * **Type**: boolean
+  * **Default value**: True
+  * **Required**: *no*
+
+* *nms_eta*
+
+  * **Description**: eta parameter for adaptive NMS.
+  * **Range of values**: a floating-point number in close range `[0, 1.0]`.
+  * **Type**: `float`
+  * **Default value**: `1.0`
+  * **Required**: *no*
+
+**Inputs**:
+
+There are 2 kinds of input formats. The first one is -
```

Comment:
nit: It would be great to emphasize the difference of these two formats by simpy describing the classes and batches being provided differently

Reply:
sure~

Reply:
@jane-intel @rkazants This is the only change compared to previous spec. All other places are exactly copied from opset8.

Reply:
add differences.